### ISSUE TITLE
Code Review 1

### ISSUE THREAD

**Issue by acomagu** [Yamashou/task-manage#3]

**Comment by acomagu** [comment]
- [add.go#L27](https://github.com/Yamashou/task-manage/blob/master/add.go#L27) `i` is usually means `iterator` so should have other name.
- [add.go#L27](https://github.com/Yamashou/task-manage/blob/master/add.go#L27) [add.go#L30](https://github.com/Yamashou/task-manage/blob/master/add.go#L30) [add.go#L31](https://github.com/Yamashou/task-manage/blob/master/add.go#L31) [add.go#L37](https://github.com/Yamashou/task-manage/blob/master/add.go#L37) [done.go#L17](https://github.com/Yamashou/task-manage/blob/master/done.go#L17) [list.go#L35](https://github.com/Yamashou/task-manage/blob/master/show.go#L35) All `os.Args` in `Run()` must be replaced `args`, the first argument of `Run()`.
- [add.go#L15](https://github.com/Yamashou/task-manage/blob/master/add.go#L15) [add.go#L16](https://github.com/Yamashou/task-manage/blob/master/add.go#L16) It's better JSON property is lowerCamelCase.
- [add.go#L46](https://github.com/Yamashou/task-manage/blob/master/add.go#L46) This line should be just next to `os.Create()` statement (and the error checking), in this case, L42.
- [add.go#L36](https://github.com/Yamashou/task-manage/blob/master/add.go#L36) It's better the data files to be placed `$HOME/.task-manage/` or `$XDG_DATA_HOME/task-manage/`(ref. [XDG Base Directory サポート - ArchWiki](https://wiki.archlinuxjp.org/index.php/XDG_Base_Directory_%E3%82%B5%E3%83%9D%E3%83%BC%E3%83%88))
- [add.go#L37](https://github.com/Yamashou/task-manage/blob/master/add.go#L37) [done.go#L16](https://github.com/Yamashou/task-manage/blob/master/done.go#L16) In some filesystem, characters that can not be used for filename are exists. Hashing or escaping(sanitizing) will be needed(and now task-name can't be include underscore, but it'll be fixed by that).
- [add.go#L37](https://github.com/Yamashou/task-manage/blob/master/add.go#L37) [done.go#L16](https://github.com/Yamashou/task-manage/blob/master/done.go#L16) Path specification can be commonize.
- [add.go#L25](https://github.com/Yamashou/task-manage/blob/master/list.go#L25) [list.go#L30](https://github.com/Yamashou/task-manage/blob/master/list.go#L30) What means `1`?
- [show.go#L22](https://github.com/Yamashou/task-manage/blob/master/show.go#L22) [list.go#L19](https://github.com/Yamashou/task-manage/blob/master/list.go#L19) They can be commonize. It's better to commonize file access over all command, for example, create `Datastore` struct and access by `Datastore.TaskNames()` or `Datastore.Get()`.
- [main.go#L10](https://github.com/Yamashou/task-manage/blob/master/main.go#L10) Split file for `Init`.

Basically it's readable and will works well.

In addition, error checking or go-fmt is missing in some points. I recommend to use [gometalinter](https://github.com/alecthomas/gometalinter), like `gometalinter --enable=gofmt --disable=vetshadow --deadline=5m --concurrency=5`.
